### DISCUSSION TITLE
🛠️ Project functions / Funciones del proyecto 🛠️

### DISCUSSION THREAD

**Discussion by Omitg24** [Arquisoft/lomap_es3a#36] [Ideas]
<p align="center">
<img src="https://user-images.githubusercontent.com/91057639/218811134-2eec75dc-a6d5-42f7-b1d6-fae6bd5ac158.png" width="640" height="320">
</p>

# EN: Discussion objective 
<p align="justify">
Here we will offer ideas about the functional aspect of the project (back-end), showing what we think about specific functions or what we would like to see implemented.
</p>
<p align="justify">
We will also ask help about something where we are blocked and don't know how to proceed.
</p>

# ES: Objetivo de la discusión
<p align="justify">
Aquí ofreceremos ideas sobre el aspecto funcional del proyecto (back-end), mostrando lo que pensamos sobre funciones específicas o lo que nos gustaría ver implementado.
</p>
<p align="justify">
También pediremos ayuda sobre algo en lo que estamos bloqueados y no sabemos cómo proceder.
</p>

**Comment by Omitg24** [comment]
# Subidas varias funciones
Hola equipo!
He subido varias cosas a mi rama, modificando bastante, os recomiendo que actualiceis vuestras ramas en base a la mía ya que ya tiene la estructura creada, por lo que puede ser interesante.

Decidme que os parece lo que he hecho y si hay algo que cambiaríais!

@UO278968 @UO278036 @uo284373

**Reply by raulEspaa** [reply]
Actualizada! Queda muy bien una interfaz muy intuitiva, cuando tenga el home os aviso

Un saludo,
Raúl.

**Reply by uo284373** [reply]
Hola, Omar!

Acabo de revisar el trabajo desarrollado en tu rama. Está muy bien implementado el diseño que habíamos fijado en los prototipos de pantalla. Trabajaré a partir de esta rama para incluir funcionalidad.

Un saludo y gracias.
Carlos

**Comment by uo284373** [comment]
# Funcionalidad para el menú lateral izquierdo 

Buenas tardes, equipo!

He estado investigando sobre como hacer el menú lateral izquierdo de la aplicación de forma que, al pulsar el botón, se despliegue el menú. La forma que he encontrado de desarrollar esto, es asignándole un método al botón que se encargue de aumentar la propiedad width del menú, si está sin desplegar, y que la disminuya, si está desplegado. Es decir, incluimos todos los componentes dentro de un div (incluyendo el botón) y mediante CSS definimos, inicialmente, la propiedad width a 0. Posteriormente, al pulsar este botón iremos cambiando el valor de esta propiedad. Además, se le puede añadir la regla "transition : 0.5s" para hacer esta transición más suave.

Espero que os sirva de ayuda para desarrollar esta funcionalidad equipo.

Un saludo.
Carlos

**Reply by Omitg24** [reply]
Muchas gracias Carlos! Lo miraré y te digo!

**Reply by raulEspaa** [reply]
Me han sido de mucha ayuda, Gracias Carlos!

**Comment by Omitg24** [comment]
# Actualización del menú lateral izquierdo
Como os he comentado, voy a cambiar el menú lateral izquierdo, haciendo que sea siempre visible, en lugar de mediante presionar el botón
